app/bundles/LeadBundle/Controller/Api/LeadApiController.php

```diff
@@ -661,6 +661,45 @@ public function getEventsAction($id)
         return $this->handleView($this->view($events));
     }
 
+    /**
+     * @param array  $parameters
+     * @param Lead   $entity
+     * @param string $action
+     *
+     * @return mixed|void
```

Comment:
Could we specify return type as array?

Reply:
Since this method is overwritten from the inherited class I don't want to change that. Maybe
```
    /**
     * {@inheritdoc}
     */
```
would be better. What do you think?

Reply:
I see. The problem itself is in the inheritance and architecture. I think we could add correct doc block, but it helps nothing in reality.